### ISSUE TITLE
Incorrect number of iterations per second

### ISSUE THREAD

**Issue by lebigot** [tqdm/tqdm#689]
- [X] I have visited the [source website], and in particular
  read the [known issues]
- [X] I have searched through the [issue tracker] for duplicates
- [X] I have mentioned version numbers, operating system and
  environment, where applicable:
  ```python
  import tqdm, sys
  print(tqdm.__version__, sys.version, sys.platform)
  ```

When setting some number of already done iterations with `initial`, **the time per iteration is very wrong**:
```python
import time
for _ in tqdm.tqdm(range(10), initial=90):  # As if we had done already 90 iterations
    time.sleep(1)  # Should show 1 it/s but shows something much faster
```
Even trying to play with the total number of iterations (even if it could be calculated as usual as `initial+len(iterable)` when possible), the time per iteration is wrong:
```python
for _ in tqdm.tqdm(range(10), initial=90, total=100):  # As if we had done already 90 iterations
    time.sleep(1)  # Should show 1 it/s but shows something much faster
```
(`total` seems to be the full number of iterations, but even using the incremental number, `total=10`, doesn't work).

Version info:

    4.24.0 2.7.13 (default, Mar  2 2018, 16:46:11) 
    [GCC 5.2.1 20150902 (Red Hat 5.2.1-2)] linux2

  [source website]: https://github.com/tqdm/tqdm/
  [known issues]: https://github.com/tqdm/tqdm/#faq-and-known-issues
  [issue tracker]: https://github.com/tqdm/tqdm/issues?q=

**Comment by MysticRyuujin** [comment]
I'm trying to figure out if this is a duplicate of what I'm seeing?

I have a list of maybe 300 items, each item takes maybe 30 seconds to process. But, when I wrap it in `for item in tqdm(myList,ascii=True):` the it/s is reported at like 15it/s...

So I'll see something like:
`30%|####.............| 83/267 [18:56<47:58, 15.73it/s]`

When in reality it's more like 0.03it/s (30 seconds)? This is my first time using tqdm so I appologize if I'm just doing it work 😄

**Comment by casperdcl** [comment]
@MysticRyuujin I think you must either have `15.73s/it` not `15.73it/s`, or you have some logic messing with `bar_format`.

**Comment by MysticRyuujin** [comment]
You, are correct...sorry!

**Comment by mtanti** [comment]
Has this been fixed please? I think it's still an issue in v4.43.0. He's a stack overflow question about it:
https://stackoverflow.com/q/62571915/652460

**Comment by casperdcl** [comment]
`tqdm.tqdm(range(10), initial=90, total=100)` should work as expected.

Seems like there's a bug with estimated `it/s` upon completion, though. Looks like a duplicate of #215.

Basically:

```python
import tqdm, time
for i in tqdm.tqdm(range(10), initial=90, total=100):
    time.sleep(0.1)  # correctly shows ~10it/s
# final state suddenly shows ~100it/s
```
